CompilerSource/main.h

```diff
@@ -8,7 +8,7 @@
  */
 
 #ifdef BUILD_DLL
-    #define DLL_EXPORT __declspec(dllexport)
+    #define DLL_EXPORT __declspec(DLLEXPORT)
```

Comment:
this is probably wrong

Reply:
Actually, I thought Robert just "fixed" this for Win64. Or WinSysWOWOMG64. Or whatever. I thought dllexport was our macro, or we had E_DLLEXPORT, or some shit.

Reply:
dllexport/dllimport are msvc things afaik

Reply:
@fundies What about this one? It's different than the one's you just (https://github.com/enigma-dev/enigma-dev/pull/2025#issuecomment-645632186) told me to replace.

Reply:
> But why did I have to go and make that one part lowercase? Also, do you still want me to make that one line lowercase I asked about?

Yes they should all be lower. I messed up in my search and replace. I was trying to rename the macro itself because in some places we defined "dllexport" (which is really hacky) but now google also redefines it causing the conflict. dllexport is c keyword on windows only so I needed you to fix that and make sure nothing was broken

Reply:
Alright 15ba9d0956dc782e2e63fb97f86ece53e7889777 should be the last of them then.